src/__tests__/getWorkspaces.test.ts

```diff
@@ -64,3 +65,33 @@ describe("getRushWorkspaces()", () => {
     ]);
   });
 });
+
+describe("getNpmWorkspaces()", () => {
+  it("gets the name and path of the workspaces", async () => {
+    const packageRoot = await setupFixture("monorepo-npm");
+    const workspacesPackageInfo = getNpmWorkspaces(packageRoot);
+
+    const packageAPath = path.join(packageRoot, "packages", "package-a");
+    const packageBPath = path.join(packageRoot, "packages", "package-b");
+
+    expect(workspacesPackageInfo).toMatchObject([
+      { name: "package-a", path: packageAPath },
+      { name: "package-b", path: packageBPath },
+    ]);
+  });
+
+  it("gets the name and path of the workspaces using the shorthand configuration", async () => {
+    const packageRoot = await setupFixture("monorepo-shorthand");
+    const workspacesPackageInfo = getYarnWorkspaces(packageRoot);
```

Comment:
Should this be `getNpmWorkspaces`?

Reply:
Yeap, good catch. Updated to `getNpmWorkspaces`